products/rhel.md

```diff
@@ -30,6 +30,15 @@ auto:
         eoes: Extended life cycle support (ELS) add-on
 
 releases:
+-   releaseCycle: "10"
+    releaseDate: 2025-05-20
+    eoas: 2030-05-31
+    eol: 2035-05-31
+    lts: 2035-05-31
```

Comment:
```suggestion
    lts: true
```
our lts tag shows when the lts release begins so this should be lts: true for this example , we use lts: date for the products like nodejs those are became lts in middle of their cycle